components/webhook-mgt/org.wso2.carbon.identity.event.publisher/src/main/java/org/wso2/carbon/identity/event/publisher/api/service/EventPublisherService.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.event.publisher.api.service;
+
+import org.wso2.carbon.identity.event.publisher.api.model.EventContext;
+import org.wso2.carbon.identity.event.publisher.api.model.SecurityEventTokenPayload;
+
+/**
+ * Service interface for topic management operations.
```

Comment:
comment not relevant

Reply:
fixed with https://github.com/wso2/carbon-identity-framework/pull/6920/commits/7a9d5c3f14773246841828e15e0e5937f1b9ea52